motioneye/locale/fr/LC_MESSAGES/motioneye.js.po

```diff
@@ -65,7 +67,7 @@ msgstr "Cela supprimera récursivement tous les fichiers du dossier cloud \""
 
 #: motioneye/static/js/main.js:827
 msgid "\", ne nur tiuj alŝutitaj de motionEye!"
-msgstr "\", pas seulement ceux téléchargés par motionEye!"
+msgstr "\", pas seulement ceux téléchargés par motionEye !"
```

Comment:
@jmichault 
Do you think it's better to have a blank here? Grammatically correct is to not have a blank between sentence and punctuation marks.

Reply:
Yes, in French double punctuation marks (like ; ! ? : ) must be preceded by a non-breaking space.  
It's specific to French.